extractors/douyin/douyin.go

```diff
@@ -53,7 +57,31 @@ func (e *extractor) Extract(url string, option extractors.Options) ([]*extractor
 		return nil, errors.WithStack(extractors.ErrURLParseFailed)
 	}
 	itemId := itemIds[len(itemIds)-1]
-	jsonData, err := request.Get("https://www.iesdouyin.com/aweme/v1/web/aweme/detail/?aweme_id="+itemId+"&aid=1128&version_name=23.5.0&device_platform=android&os_version=2333", url, nil)
+
+	// init JavaScripts runtime
+	vm := goja.New()
+	// read sign file
+	script, err := os.ReadFile("./script/douyin-sign.js")
```

Comment:
We can't use a relative path here, it won't work in binary mode, you can use the `go:embed` feature here.